assignments/objects.js

```diff
@@ -18,27 +18,49 @@ const example = {
 }
 
 // Write your intern objects here:
-
+const interns = [
+  {"id":1, "name": "Mitzi", "email": "mmelloy0@psu.edu", "gender": "F"},
+  {"id":2, "name": "Kennan", "email": "kdiben1@tinypic.com", "gender": "M"},
```

Comment:
```suggestion
  {"id":2, "name": "Kennan", "email": "kdiben1@tinypic.com", "gender": "M", speak: function() {
    return `Hello, my name is ${this.name}!`;
  }},
```
You can also declare the method in the object here rather than below. This is just a little bit neater for the next developer, to have everything in the same place